.github/pull_request_template.md

```diff
@@ -3,10 +3,9 @@
 Please provide a paragraph or two giving a summary of the change, including relevant motivation and context.
 
 # Checklist:
-
-- [ ] I have reviewed my diff in github, line by line.
+Remove the checklist to signal you've completed it. Enable auto-merge if the PR is ready to merge.
+- [ ] If the pull request requires a cryptography review (e.g. cryptographic algorithm implementations) I have added the 'crypto' tag.
+- [ ] I have reviewed my diff in github, line by line and removed unexpected formatting changes, testing logs, or commented-out code.
 - [ ] Every change is related to the PR description.
-- [ ] I have [linked](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) this pull request to the issue(s) that it resolves.
-- [ ] There are no unexpected formatting changes, superfluous debug logs, or commented-out code.
-- [ ] The branch has been merged or rebased against the head of its merge target.
-- [ ] I'm happy for the PR to be merged at the reviewer's next convenience.
+- [ ] I have [linked](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) this pull request to relevant issues (if any exist).
+- [ ] The branch is up-to-date (i.e. no Update Branch button).
```

Comment:
Feel like this one doesn't make much sense, since it can be completed once you get the PR ready, but can fall out of date pretty quickly. But I'm fine keeping it for historic reasons.